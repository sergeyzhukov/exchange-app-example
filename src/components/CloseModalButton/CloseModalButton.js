
import React from 'react'
import { Button } from 'react-native'

export default ({ onPress }) => (
  <Button
    onPress={onPress}
    title="Cancel"
    color="#4072B8"
  />
)
