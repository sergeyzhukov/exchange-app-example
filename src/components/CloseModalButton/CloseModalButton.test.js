import React from 'react'
import renderer from 'react-test-renderer'
import CloseModalButton from './CloseModalButton'

test('renders correctly', () => {
  const component = <CloseModalButton onPress={() => {}} />
  const tree = renderer.create(component).toJSON()
  expect(tree).toMatchSnapshot()
})
