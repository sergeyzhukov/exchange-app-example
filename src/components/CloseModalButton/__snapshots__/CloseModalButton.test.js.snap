// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityRole="button"
  accessibilityStates={Array []}
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <View
    style={
      Array [
        Object {},
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#007AFF",
            "fontSize": 18,
            "padding": 8,
            "textAlign": "center",
          },
          Object {
            "color": "#4072B8",
          },
        ]
      }
    >
      Cancel
    </Text>
  </View>
</View>
`;
